conftest.py

```diff
@@ -1,34 +0,0 @@
-# Licensed under a 3-clause BSD style license - see LICENSE.rst
```

Comment:
Yeah, tox needs it if you want to see pretty test headers in the CI logs. But then again, I had to remove for a couple of projects with C-extensions because of astropy/pytest-astropy-header#41

Reply:
Restored